endpoint/endpointComp/Component.cdef

```diff
@@ -0,0 +1,146 @@
+sources:
+{
+    ${CURDIR}/../endpoint_core.c
+
+    ${CURDIR}/../../output_base/external/http_parser/http_parser.c
+
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/aes.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/aesni.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/arc4.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/aria.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/asn1parse.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/asn1write.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/base64.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/bignum.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/blowfish.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/camellia.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ccm.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/certs.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/chacha20.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/chachapoly.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/cipher.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/cipher_wrap.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/cmac.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ctr_drbg.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/debug.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/des.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/dhm.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ecdh.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ecdsa.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ecjpake.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ecp.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/ecp_curves.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/entropy.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/entropy_poll.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/error.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/gcm.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/havege.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/hkdf.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/hmac_drbg.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/md2.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/md4.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/md5.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/md.c
+    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/md_wrap.c
```

Comment:
Do we really need every source file in `mbedtls`? For example, `md*.c` seem to be less useful. They can be omitted if the configuration is tweaked.

Reply:
I remove the redundant source code, which includes `md4.c`.